engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -2092,6 +2068,35 @@ public void populateDAG(LogicalPlan dag)
 
   }
 
+  private void populateOperators(AppConf appConf, LogicalPlan dag, Map<OperatorConf, Operator> nodeMap)
```

Comment:
Original function (populateDAG) was long (> 100 lines) , while adding processing for modules it will get even bigger, hence moved the code around.